index.html

```diff
@@ -280,16 +288,129 @@
             margin: auto;
         }
         @media screen and (max-width: 768px) {
-            .grid-container {
+           .grid-container {
                 grid-template-columns: 100%;
                 flex-direction: column;
+            }
+
+            .why-us .grid-container {
                 padding: 0px 40px 73px 40px;
-                
             }
-            .section_title {
+
+            .why-us .section_title {
                 padding: 75px 40px 45px ;
                 flex-direction: column;
                 align-items: center;
+                text-align: center;
+            }
+        }
+        .clients-say {
+            background: var(--dark-background-color, #252B42);
+            font-family: 'Montserrat';
+            display: flex;
+            padding: 0px 195px;
+            justify-content: center;
+            align-items: center;
+            align-self: stretch;
+        }
+
+        .clients-say .container {
```

Comment:
I bet you don't need to overwrite all the styles for container  for "client-say" block. If there is any difference, overwrite only it. If there is no difference in container style, just use class "container" and no need to add any other style there

Reply:
hello Ivan, thank you for your feedback , I apllied the adjustments you suggested